Danki2/Assets/Scripts/AI/Behaviour.cs

```diff
@@ -0,0 +1,4 @@
+﻿public interface Behaviour<T> where T : Actor
```

Comment:
C#/Unity convention is to prefix interfaces with an "I", I quite like this because you can differentiate them from classes in the file explorer and you can leave the non prefixed name for a standard implementation (in some cases).

Reply:
Yeah I like this too, I've changed this interface and the one you commented on below